do-server/src/index.mjs

```diff
@@ -1,5 +1,7 @@
 import { mutators } from '../../syncosaurus/mutators.js';
+import { authHandler } from '../../syncosaurus/authHandler.js';
 
+// TODO: This needs to be moved to be an environment variable
```

Comment:
This could live in the new syncosaurus.json configuration file... our current codebase doesn't implement that file but with the new CLI it starts to reference that for certain config things. This could be a really good config variable there!